### ISSUE TITLE
cate-2.0.0.dev16 fails to start due to unable to find dll files

### ISSUE THREAD

**Issue by Evadzi** [CCI-Tools/cate#725]
### Expected behavior

*I installed cate-2.0.0.dev16 on my windows 10 desktop PC. I expect the software to open when double clicked*

### Actual behavior

*Cate was unable to launch and returns an error message (see pix below)*

![cate_failed_to_start](https://user-images.githubusercontent.com/37798521/43830207-0c4c0b34-9b01-11e8-9048-40a39c475a7d.PNG)

The issue is difficult to reproduce but a familiar issue @JanisGailis 

### Hints
1. *the issue is about the fiona module(refer to the log file below* 

details...
```
[2018-08-08 11:23:04.232] [error] Invalid PID: undefined
[2018-08-08 11:23:04.232] [info] Cate service parent process is no longer connected.
[2018-08-08 11:23:04.232] [info] Main window closed.
[2018-08-08 11:23:06.001] [info] ===============================================================================
[2018-08-08 11:23:06.017] [info] Cate Desktop, version 2.0.0-dev.16
[2018-08-08 11:23:06.017] [info] ===============================================================================
[2018-08-08 11:23:06.017] [info] process.versions = { http_parser: '2.7.0',
  node: '8.2.1',
  v8: '5.9.211.38',
  uv: '1.13.1',
  zlib: '1.2.11',
  ares: '1.10.1-DEV',
  modules: '57',
  openssl: '1.0.2l',
  electron: '1.8.4',
  chrome: '59.0.3071.115',
  'atom-shell': '1.8.4' }
[2018-08-08 11:23:06.017] [warn] debugWorldView = undefined
[2018-08-08 11:23:06.017] [info] process.env.NODE_ENV = production
[2018-08-08 11:23:06.017] [info] appPath: C:\Users\prosper\AppData\Local\Programs\cate-desktop\resources\app.asar
[2018-08-08 11:23:06.017] [info] appConfig: { webAPIConfig: 
   { servicePort: 9090,
     serviceAddress: '',
     serviceFile: 'C:\\Users\\prosper\\.cate\\webapi-info.json',
     processOptions: {} } }
[2018-08-08 11:23:06.048] [info] Ready.
[2018-08-08 11:23:06.444] [info] Splash says: Waiting for Cate service...
[2018-08-08 11:23:06.445] [info] Waiting for response from Cate service http://localhost:9090/
[2018-08-08 11:23:07.464] [info] No response from Cate service after 1.02 seconds: Error: connect ECONNREFUSED 127.0.0.1:9090
[2018-08-08 11:23:07.465] [info] Splash says: Waiting for Cate service (1.02s)
[2018-08-08 11:23:07.469] [info] Checking Cate installation directory: C:\Users\prosper\.cate\2.0.0.dev16\cate.location
[2018-08-08 11:23:07.471] [info] Cate WebAPI executable found: C:\Users\prosper\cate-2.0.0.dev16\Scripts\cate-webapi-start.exe
[2018-08-08 11:23:07.472] [info] Computed setup information:  { oldCateDir: 'C:\\Users\\prosper\\cate-2.0.0.dev16',
  newCateDir: 'C:\\Users\\prosper\\cate-2.0.0.dev16',
  oldCateVersion: '2.0.0.dev16',
  newCateVersion: '2.0.0.dev16',
  setupReason: null }
[2018-08-08 11:23:07.473] [info] Cate directory set to C:\Users\prosper\cate-2.0.0.dev16
[2018-08-08 11:23:07.474] [info] Splash says: Searching unused port...
[2018-08-08 11:23:07.485] [info] Starting Cate service: "C:\Users\prosper\cate-2.0.0.dev16\Scripts\activate" "C:\Users\prosper\cate-2.0.0.dev16" & cate-webapi-start --caller cate-desktop --port 9090 --file "C:\Users\prosper\.cate\webapi-info.json" --verbose
[2018-08-08 11:23:07.502] [info] Cate service started (pid=7016).
[2018-08-08 11:23:08.004] [info] Waiting for response from Cate service http://localhost:9090/
[2018-08-08 11:23:09.008] [info] No response from Cate service after 2.56 seconds: Error: connect ECONNREFUSED 127.0.0.1:9090
[2018-08-08 11:23:09.008] [info] Splash says: Waiting for Cate service (2.56s)
[2018-08-08 11:23:09.509] [info] Waiting for response from Cate service http://localhost:9090/
[2018-08-08 11:23:10.507] [error] cate-webapi: Traceback (most recent call last):
  File "C:\Users\prosper\cate-2.0.0.dev16\Scripts\cate-webapi-start-script.py", line 6, in <module>
    from cate.webapi.start import main
  File "C:\Users\prosper\cate-2.0.0.dev16\lib\site-packages\cate\webapi\start.py", line 54, in <module>
    from cate.core.types import ValidationError
  File "C:\Users\prosper\cate-2.0.0.dev16\lib\site-packages\cate\core\__init__.py", line 41, in <module>
    from .ds import DataStore, DataSource, open_dataset, find_data_sources, DATA_STORE_REGISTRY
  File "C:\Users\prosper\cate-2.0.0.dev16\lib\site-packages\cate\core\ds.py", line 90, in <module>
    from .cdm import Schema, get_lon_dim_name, get_lat_dim_name
  File "C:\Users\prosper\cate-2.0.0.dev16\lib\site-packages\cate\core\cdm.py", line 116, in <module>
    from .opimpl import get_lat_dim_name_impl, get_lon_dim_name_impl
  File "C:\Users\prosper\cate-2.0.0.dev16\lib\site-packages\cate\core\opimpl.py", line 33, in <module>
    from .types import PolygonLike, ValidationError
  File "C:\Users\prosper\cate-2.0.0.dev16\lib\site-packages\cate\core\types.py", line 44, in <module>
    import geopandas
  File "C:\Users\prosper\cate-2.0.0.dev16\lib\site-packages\geopandas\__init__.py", line 4, in <module>
    from geopandas.io.file import read_file
  File "C:\Users\prosper\cate-2.0.0.dev16\lib\site-packages\geopandas\io\file.py", line 3, in <module>
    import fiona
  File "C:\Users\prosper\cate-2.0.0.dev16\lib\site-packages\fiona\__init__.py", line 69, in <module>
    from fiona.collection import Collection, BytesCollection, vsi_path
  **File "C:\Users\prosper\cate-2.0.0.dev16\lib\site-packages\fiona\collection.py", line 9, in <module>
    from fiona.ogrext import Iterator, ItemsIterator, KeysIterator
ImportError: DLL load failed: Das angegebene Modul wurde nicht gefunden.**
```

### Specifications 

*cate-2.0.0.dev16, Windows 10*

**Comment by hans-permana** [comment]
this was caused by gdal and fiona relying on vs2015_runtime 14 and in new Miniconda, v15 is automatically installed. 

A workaround was done in this commit https://github.com/CCI-Tools/cate-conda/commit/2599d8ea4d8cd28da8aa295163ef1b153e4593d2 by adding `vs2015_runtime version 14` specifically in cate-cli `meta.yml`. This seems to fix the problem.

**Comment by hans-permana** [comment]
The root cause of this issue: https://github.com/conda/conda/issues/7626

**Comment by Evadzi** [comment]
Issue solved in cate-2.0.0.dev17
